src/components/content-tab-settings.tsx

```diff
@@ -21,7 +21,7 @@ interface ContentTabSettingsProps {
 function SettingsRow( { children, label }: PropsWithChildren< { label: string } > ) {
 	return (
 		<tr className="align-top">
-			<th className="text-nowrap text-a8c-gray-50 pb-4 ltr:pr-6 rtl:pl-6 ltr:text-left rtl:text-right font-normal">
+			<th className="text-nowrap text-a8c-gray-50 pb-4 ltr:pr-10 rtl:pl-10 ltr:text-left rtl:text-right font-normal">
```

Comment:
We could instead use padding-inline-end:

```suggestion
			<th className="text-nowrap text-a8c-gray-50 pb-4 pe-10 ltr:text-left rtl:text-right font-normal">
```

Reply:
Thanks, changed in 6fdefb1501ac1de6286b934eeaa12afae08af609